default/col_buildings.txt

```diff
@@ -184,7 +184,7 @@ BuildingType
             activation = Turn low = Source.CreationTurn + 2
             effects = Destroy
     ]
-    icon = "icons/building/generic_building.png"
+    icon = "icons/species/abaddonnian.png"
```

Comment:
I have no opinion, it wasn't something I cared to do and there are many other instances where icons in one folder are used for other things, and not just coded up from me. If you want to do it feel free but there are multiple other instances and I don't see the need to duplicate files and increase download size/install time for little if any net gain.